File: 9/9.cs
public class Solution {
    public bool IsPalindrome(int x) {
        string strNum = Convert.ToString(x);
        string newNum = "";

        for (int i = strNum.Length-1; i >= 0; i--)
        {
            newNum += strNum[i];
        }

        if (newNum == strNum)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

File: 9/9.java
class Solution {
    public boolean isPalindrome(int x) {
        String strNum = String.valueOf(x);
        String strRevNum = "";
        for (int i = strNum.length()-1; i >= 0; i--) {
            strRevNum += strNum.charAt(i);
        }
        
        if (strNum.equals(strRevNum)) {
            return true;
        }
        else {
            return false;
        }
    }
}
